templates/e-commerce/api/high-low-all-products.json

```diff
@@ -1,88 +1,88 @@
 {
```

Comment:
Can you make the script - output that as a comment on the first line of this file.